python/cudf/cudf/core/dataframe.py

```diff
@@ -1551,7 +1551,15 @@ def _concat(
         # Reassign precision for any decimal cols
         for name, col in out._data.items():
             if isinstance(col, cudf.core.column.Decimal64Column):
-                col = col._with_type_metadata(tables[0]._data[name].dtype)
+                out._data[name] = col._with_type_metadata(
+                    tables[0]._data[name].dtype
+                )
+            elif isinstance(col, cudf.core.column.StructColumn):
```

Comment:
I can't make a suggestion where I want, but you can combine these with:
```python
isinstance(col, (cudf.core.column.Decimal64Column, cudf.core.column.StructColumn)):
```

https://docs.python.org/3/library/functions.html#isinstance
> If classinfo is a tuple of type objects (or recursively, other such tuples) or a Union Type of multiple types, return True if object is an instance of any of the types.

Reply:
Done 👍